core/src/main/scala/cats/data/NonEmptySet.scala

```diff
@@ -0,0 +1,410 @@
+/*
+ * Copyright (c) 2018 Luka Jacobowitz
```

Comment:
Is there a reason that this file has this header but `Newtype.scala` doesn't?

Reply:
Nope, just copy and paste, Kai suggested we try to standardize on something in a future PR, possibly with sbt-header or something like that :)